src/output-filestore.c

```diff
@@ -166,8 +166,12 @@ static void OutputFilestoreFinalizeFiles(ThreadVars *tv, const OutputFilestoreLo
                     ff->file_store_id) == (int)sizeof(js_metadata_filename)) {
             WARN_ONCE(WOT_SNPRINTF, "Failed to write file info record. Output filename truncated.");
         } else {
+            // Set temporarily flag that file is stored for logging.
```

Comment:
can we get rid of this hack?

Reply:
Looks like unneeded thanks to `OUTPUT_FILEDATA_FLAG_CLOSE guarding the call  on OutputFilestoreFinalizeFiles